index.js

```diff
@@ -799,8 +799,8 @@ app.post('/smokeTest/setup', async (request, response) => {
       3000,
       recipientNumber,
       twilioNumber,
-      recipientNumber,
-      9000,
+      [recipientNumber],
+      45000,
```

Comment:
I changed this because I wasn't getting the fallback message when running the smoketest. It needed to be a value greater than the reminder timeout (which is `30000`).